kview/src/main/java/dev/ikm/komet/kview/mvvm/view/properties/PropertiesController.java

```diff
@@ -148,12 +153,19 @@ public class PropertiesController implements Serializable {
 
     private boolean hasOtherNames = false;
 
+    private StampAddFormViewModel stampAddFormViewModel;
+
+    private StampCreateFormViewModel stampCreateFormViewModel;
+
+    private boolean editMode;
 
     public PropertiesController() {
     }
 
     public PropertiesController(UUID conceptTopic) {
         this.conceptTopic = conceptTopic;
+        this.stampAddFormViewModel = new StampAddFormViewModel(CONCEPT);
+        this.stampCreateFormViewModel = new StampCreateFormViewModel(CONCEPT);
```

Comment:
(Optional) Can these be injected? You don't have to just thought it might be more convenient and less code.

Reply:
@carldea I have 2 view model classes and injection would only work for 1. At this point I don't know exactly which view model is going to be used.